### PR TITLE
Adding osbs labels to cli artifacts image

### PR DESCRIPTION
**Pull request by mohammedzee1000**
These labels are required by osbs.
Signed-off-by: Mohammed Zeeshan Ahmed <mohammed.zee1000@gmail.com>

### CODE REVIEW

File: Dockerfile.rhel [hunk 1]
```diff
@@ -8,7 +8,16 @@ WORKDIR /go/src/github.com/openshift/odo
...
-LABEL com.redhat.component=atomic-openshift-odo-cli-artifacts
+
+LABEL com.redhat.component=atomic-openshift-odo-cli-artifacts-container \ 
+    name=openshift/odo-cli-artifacts \ 
+    io.k8s.display-name=atomic-openshift-odo-cli-artifacts-image \
+    maintainer=devtools-deploy@redhat.com \ 
+    summary="This image containing the Linux, Mac and Windows version of odo"
+
+# Change version as needed. Note no - is allowed
+LABEL version=1.0.0beta5
```
Comment (mik-dass) [inline Dockerfile.rhel]: Does this need to be changed with every release? If yes then let's add it to the docs

File: Dockerfile.rhel [hunk 2]
```diff
@@ -8,7 +8,16 @@ WORKDIR /go/src/github.com/openshift/odo
...
 FROM registry.access.redhat.com/ubi7/ubi
-LABEL com.redhat.component=atomic-openshift-odo-cli-artifacts
+
+LABEL com.redhat.component=atomic-openshift-odo-cli-artifacts-container \ 
+    name=openshift/odo-cli-artifacts \ 
+    io.k8s.display-name=atomic-openshift-odo-cli-artifacts-image \
+    maintainer=devtools-deploy@redhat.com \ 
+    summary="This image containing the Linux, Mac and Windows version of odo"
+
+# Change version as needed. Note no - is allowed
```
Comment (amitkrout) [inline Dockerfile.rhel]: ```-``` -> ```"-"``` 
makes more sense i guess

File: Dockerfile.rhel [hunk 3]
```diff
@@ -8,7 +8,16 @@ WORKDIR /go/src/github.com/openshift/odo
 RUN make cross
 
 FROM registry.access.redhat.com/ubi7/ubi
-LABEL com.redhat.component=atomic-openshift-odo-cli-artifacts
+
+LABEL com.redhat.component=atomic-openshift-odo-cli-artifacts-container \ 
+    name=openshift/odo-cli-artifacts \ 
+    io.k8s.display-name=atomic-openshift-odo-cli-artifacts-image \
+    maintainer=devtools-deploy@redhat.com \ 
+    summary="This image containing the Linux, Mac and Windows version of odo"
```
Comment (mik-dass) [inline Dockerfile.rhel]: "This image contains the Linux...."?

Comment (mohammedzee1000) [reply Dockerfile.rhel]: Thanks, this was carry forward from copy-paste

File: scripts/bump-version.sh [hunk 4]
```diff
@@ -37,3 +37,7 @@ echo "**************************************************************************
 echo "****************************************************************************************"
 echo "* Don't forget to update build/VERSION once the binaries become available !            *"
 echo "****************************************************************************************"
+
+echo "****************************************************************************************"
+echo "* Don't forget to update Dockerfile.rhel version= label !            *"
```
Comment (amitkrout) [inline scripts/bump-version.sh]: +100
/lgtm

**Pull request merged**